assets/scss/modules/typography/_lists.scss

```diff
@@ -1,17 +1,17 @@
 ul, ol {
   margin: $leading 0 $leading*2 0;
-  padding-left: 1em;
-  list-style-type: square;
+  padding-left: 0px;
+  list-style-type: none;
```

Comment:
Oh, this as well. This is zeroing out all lists? I'd suggest setting it to `disc` but that's a personal opinion.